### ISSUE TITLE
Page Redirect To Welcome to Redux Framework

### ISSUE THREAD

**Issue by adrian200065** [reduxframework/redux-framework#3338]
Hello,

I have seemed other people mentioned this issue but no real solution. I think in some cases you were asking for the information to log in to their websites to see the issue but did not get the log in information. If you need my log in I will give it. The issue is when I log in or click "Visit site" I am redirecting to Redux Welcome page. Here is the last part of the URL wp-admin/tools.php?page=redux-about.

Thanks.

**Comment by adrian200065** [comment]
I forgot to mention I am using version 3.6.

**Comment by kprovance** [comment]
What you have described in a premium service now.

If you are having an issue, please click here (https://github.com/reduxframework/redux-framework/issues/new) review all the provided text and reply with all the required information.

**Comment by adrian200065** [comment]
I just want to restate the issue to make there is not misunderstanding what I am asking.  The issue is I can't go to my website when I log in because when I try to go to the website I am being redirected back to Redux Framework Welcome page.  So are saying to get support for this issue I need premium service?

**Comment by kprovance** [comment]
No, for us to log into your website to debug is a premium service.  

So, you go to your website wwww.<whatever>.com and you see a redux welcome page?  This is what I am taking away from your description, which makes no sense.

Also, we need your support has/URL, where, if you are in fact seeing a welcome page, should be under the support tab.

Also, with ach minor version, the welcome page does come up on first launch.  That is by design. After that page comes up, go to another one.

**Comment by adrian200065** [comment]
Ok. I understand know.
Yes, if I am login and I go to www.whatever.com I see redux welcome page instead of my website.  The redux welcome page is located under the tool admin menu and yes when I log in I am taking immediately to the redux welcome page. In my dev environment, it works great then I uploaded it to a live environment and that is when I see the issue. The only change I made is moving it from dev to live environment.

Let me do some more debugging and see if I can figure it out. If not I will sign up for your premium service. 

Thank you for your response.

**Comment by kprovance** [comment]
I am not understanding, you are describing the backend of WordPress, when www.whatever.com is the frontend.  So, which is it?  The frontend should never see any welcome page.

So, when you login to the backend, and Ill just assume backend at this point, you see the welcome page.  Again, after a minor upgrade, expected.  CLick Pages, or Posts, or some other item on the admin bar, what happens then?  And could I PLEASE get that support URL/hash?  I've asked for it twice now.

**Comment by adrian200065** [comment]
If I am log out I don't have any issue. When I am login to the backend as an admin. In the upper left corner where the website name is when I click on "Visit Site," I am redirected/taken to Redux welcome page.

**Comment by kprovance** [comment]
That's not what I asked.  I said "Pages" or "Posts".  And I need that support URL, please.

**Comment by adrian200065** [comment]
This is the URL http://theme-dev.dominiongraphicdesign.com.

**Comment by adrian200065** [comment]
Sorry here is the correct URL http://theme-dev.dominiongraphicdesign.com/wp-admin/tools.php?page=redux-support

**Comment by adrian200065** [comment]
And again I mistake... the first time using this... Support URL
http://support.redux.io/?id=5%2FPdL4wWEW1Y%2BX0C0xndl8hC6gYtSunnTyWXc3zXbOYs0u8NhNg4gMOEF%2F4WcSdXcB7ga8DD6x7zNnrL6kLEc2hi8ZJvelxRpvf6tFHga9Pb3GLjmmBL3cqTFVl7bbBL

**Comment by kprovance** [comment]
Instead of this:
```
http://theme-dev.dominiongraphicdesign.com/wp-admin/tools.php?page=redux-support
```

Use this:
```
http://theme-dev.dominiongraphicdesign.com/wp-admin
```

Anytime you try to logon with that tools parameter, you will see the welcome page.

**Comment by adrian200065** [comment]
That us what I used to log in http://theme-dev.dominiongraphicdesign.com/wp-admin. I just sent you the wrong support URL.

**Comment by kprovance** [comment]
Okay, you're not understanding me, and I'm not sure what else to do here.

Are you the theme author?  Or a user of this theme?

**Comment by adrian200065** [comment]
I am the theme author. Thank you for help so far. I will keep doing some debugging to try to resolve the issue. I have been disabling plugins and functions to see if there is a conflict but so far none. I just don't understand why the issue is not in my local dev environment but I uploaded the same theme to my live dev environment the issue arise. Again thanks for your help.

**Comment by kprovance** [comment]
Because it sounds like you are trying to login with the tools parameter attached to the URL.

Look, clear your browsing history, the URL history, cookies, everything.

Then click this: http://theme-dev.dominiongraphicdesign.com/wp-admin (without the 'tools' parameter.

**Comment by adrian200065** [comment]
I think I found the issue. I have a function that redirects the user back to home page after log off but it seems like it was affecting the backend. I disabled the function thought I was clearing my browser cache. So base on your suggestion I use a different browser and log in and did not see.  The function is a function I used on a few of websites not sure why there is an issue with this website.  Also, this is the first time I am using Redux Framework. 

Thanks again kprovance.
